bundle/app/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -93,15 +93,9 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    categories: Application Runtime
-    containerImage: quay.io/kiegroup/kogito-serverless-operator:0.0.0
-    description: SonataFlow Kubernetes Operator for deploying workflow applications
-      based on the CNCF Serverless Workflow specification
     operators.operatorframework.io/builder: operator-sdk-v1.25.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-    repository: https://github.com/kiegroup/kogito-serverless-operator
-    support: Red Hat
-  name: sonataflow-operator.vlatest
+  name: sonataflow-operator.v2.0.0-snapshot
```

Comment:
Please revert these annotations.

Reply:
Ok, unfortunately [this](https://github.com/kiegroup/kogito-serverless-operator/commit/ab7f9cb61c537cc0a0b9ebd24c168d70db5f1a9a) was merged after the rebase with main and I've lost that changes